tests/foreman/cli/test_computeresource.py

```diff
@@ -291,3 +291,73 @@ def test_update_negative(self, options):
         options.pop('new-name', None)
         for key in options.keys():
             self.assertEqual(comp_res[key], result.stdout[key])
+
+    @data('true', 'false')
+    def test_set_console_password_v1(self, set_console_password):
+        """@Test: Create a compute resource with ``--set-console-password``.
+
+        @Feature: Compute Resource
+
+        @Assert: The option is accepted, and the passed-in value can be read
+        back.
+
+        Targets BZ 1100344 and 1200098.
+
+        """
+        name = gen_string('utf8')
+        result = ComputeResource.create({
+            'name': name,
+            'provider': 'Libvirt',
+            'set-console-password': set_console_password,
+            'url': gen_url(),
+        })
+        self.assertEqual(result.return_code, 0)
+        self.assertEqual(len(result.stderr), 0)
+
+        if bz_bug_is_open(1200098):
+            return
```

Comment:
What is the reason for the pass instead of a skip? Wouldn't be better to ensure the entire test passing? Otherwise this use case could not be done by a customer.